app/stores/AuthStore.js

```diff
@@ -36,10 +36,17 @@ export default class AuthStore {
   @observable suspendedContactEmail: ?string;
   @observable config: ?Config;
   rootStore: RootStore;
+  externalAuthCookie: ?string;
 
   constructor(rootStore: RootStore) {
     this.rootStore = rootStore;
 
+    // attempt to fetch an authorization cookie potentially set by an external
+    // authorizing proxy (if configured), to use with ApiClient
+    if (env && env.EXTERNAL_AUTH_COOKIE) {
+      this.externalAuthCookie = getCookie(env.EXTERNAL_AUTH_COOKIE);
+    }
```

Comment:
I don't think this needs to be in the auth store to be honest, we can evaluate it once in the context of the ApiClient module on load, I'd also be totally fine with `CF_Authorization` being the default, it's never come up before so the need for it being configurable seems slim.

How does that sound?

Reply:
The motivation to use the auth store was that it's already imported and used in ApiClient and this could also be useful for possible future integration of external authorization with Outline authorization.
Though I agree that's a kind of over-engineering.

Reply:
Always a balancing act, particularly in long running projects like this I'm wary of introducing config that I don't know if it's used but have to support forever 😅